### PR TITLE
Update manual run page

### LINKED ISSUE #2593: Update manual run page
# Refactor

Manual run page is outdated. We now have the ability to provide multiple servers and the current page doesn't document the options well (what is the second dropdown, what does it do? Which should I pick?)
image

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes #2593 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [ ] ~Was the documentation framework updated to reflect the changes?~
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [ ] ~Do all unit tests pass?~
* [ ] ~Do all end-to-end tests pass?~
* [x] Any other testing performed?
    > Tested by running the Island
* [x] If applicable, add screenshots or log transcripts of the feature working

image

### CODE REVIEW

**Review by ilija-lazoroski — CHANGES_REQUESTED** [review]
I think this issue was primarily to add the list of IPs as list of servers to the command.

**Comment by mssalvatore** [comment]
> I think this issue was primarily to add the list of IPs as list of servers to the command.

+1

These changes are good. The clarification is needed. But we want to have all Island IPs in the agent command's server list.

**Comment by shreyamalviya** [comment]
image

**Review by ilija-lazoroski — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Hard-coded Mimikatz credentials collector to a plugin. #3168
 - Hard-coded Zerologon exploiter to a plugin. #3164
 - Hard-coded SSH credentials collector to a plugin. #3169
+- Manual run page UI. #3328
```
Comment (mssalvatore) [inline CHANGELOG.md:22]: Use the issue number, not the PR number.

**Pull request merged**